global.R

```diff
@@ -44,7 +44,7 @@ shhh(library(scales))
 
 shhh(library(htmltools))
 
-
+source("R/ui_panels/accessibility_statement.R")
```

Comment:
Could do this to then cover any other files added into that folder?

`lapply(list.files("R/ui_panels/", full.names = TRUE, recursive = TRUE), source)`

Reply:
Done